coderd/database/dbtestutil/postgres.go

```diff
@@ -93,7 +93,7 @@ func initDefaultConnection(t TBSubset) error {
 		if !containsAnySubstring(errString, retryableErrSubstrings) {
 			break
 		}
-		t.Logf("failed to connect to postgres, retrying: %s", errString)
+		t.Logf("%s failed to connect to postgres, retrying: %s", time.Now().Format(time.StampMilli), errString)
```

Comment:
added this because it looks like we only retried 3 times, even though we're supposed to wait 10 seconds. Timestamp information will be helpful if we hit it again.